docs/source/tutorials/quick_start.rst

```diff
@@ -0,0 +1,68 @@
+===============
+ALE Quick Start
+===============
+
+This document provides a set of steps to get setup for generating an Image Support
+Data (ISD) for an image.
```

Comment:
I'd say either "generating an Image Support Data (ISD) file" or omit the "an" and just say "generating Image Support Data (ISD)" for an image.